packages/grafana-data/src/transformations/transformers/concat.ts

```diff
@@ -73,7 +73,7 @@ export function concatenateFields(data: DataFrame[], opts: ConcatenateTransforme
         } else if (opts.frameNameMode === ConcatenateFrameNameMode.Label) {
           copy.labels = { ...f.labels };
           copy.labels[frameNameLabel] = frame.name;
-        } else if (!copy.name || copy.name === 'Value') {
+        } else if (!copy.name || copy.name === TIME_SERIES_VALUE_FIELD_NAME) {
```

Comment:
kinda unrelated... but it took me a while to track down why stuff was behaving weird with "value" rather than "Value"